src/nupic/os/Timer.cpp

```diff
@@ -66,38 +66,6 @@ static UInt64 getCurrentTime()
   return (UInt64)(v.QuadPart) - initialTicks_;
 }
 
-#elif defined(NTA_PLATFORM_darwin)
```

Comment:
This flag never existed. I'm removing it because Darwin works well using the same for Linux..